kore/src/Kore/Internal/InternalMap.hs

```diff
@@ -45,26 +45,26 @@ type MapElement = Element NormalizedMap
 
 -- | Wrapper for normalized maps, to be used in the `builtinAcChild` field.
 newtype NormalizedMap key child = NormalizedMap {getNormalizedMap :: NormalizedAc NormalizedMap key child}
-    deriving (Eq, Ord, Show)
-    deriving (Foldable, Functor, Traversable)
-    deriving (GHC.Generic)
+    deriving stock (Eq, Ord, Show)
+    deriving stock (Foldable, Functor, Traversable)
+    deriving stock (GHC.Generic)
     deriving anyclass (Hashable, NFData)
     deriving anyclass (SOP.Generic, SOP.HasDatatypeInfo)
     deriving anyclass (Debug, Diff)
 
 instance AcWrapper NormalizedMap where
     newtype Value NormalizedMap child = MapValue {getMapValue :: child}
-        deriving (Eq, Ord, Show)
-        deriving (Foldable, Functor, Traversable)
-        deriving (GHC.Generic)
+        deriving stock (Eq, Ord, Show)
+        deriving stock (Foldable, Functor, Traversable)
```

Comment:
Switching to `newtype` deriving gives:
```
    • No instance for (Traversable (Value NormalizedMap))
        arising from the first field of ‘NormalizedMap’
          (type ‘NormalizedAc NormalizedMap key child’)
      Possible fix:
        use a standalone 'deriving instance' declaration,
          so you can specify the instance context yourself
    • When deriving the instance for (Traversable (NormalizedMap key))
```
The error is the same for `Foldable` and `Functor`.